### PR TITLE
[KN] add footers,json format,charts

### PR DESCRIPTION
**Pull request by bestnewkevin**
Created FullChart Component which holds a chart and the filter dropdowns for the chart which makes multi chart happen

### CODE REVIEW

File: plutus-frontend/src/recommendations.js [hunk 1]
```diff
@@ -6,6 +6,26 @@ import Form from 'react-bootstrap/Form'
...
+  {
+    id: '3 - Kramer',
+    name: 'Kramer'    
+  },
+  {
+    id: '4 - George',
+    name: 'George'    
+  },
+];
+
```
Comment (dnntrn) [inline plutus-frontend/src/recommendations.js]: What is this for? It doesn't seem to be used in other parts of this file.

File: plutus-frontend/src/components/FullChart.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+   
+    render() { 
+        return ( 
+            <div>
+            <Row>
+              <Col>
+                <div id="filterBox">
+                <Dropdown type="Company" callback={this.companyCallback} opts={this.props.ddopts} />
+                <Dropdown type="Level" callback={this.levelCallback} opts={this.props.levelsopts} />
```
Comment (dnntrn) [inline plutus-frontend/src/components/FullChart.js]: what is ddopts? Or levelsopts? What do they stand for?

**Comment by dnntrn** [comment]
Please add a description to explain what this PR is. Also, include screenshots of your changes.

File: plutus-frontend/src/recommendations.js (hunk 1 above)
Comment (bestnewkevin) [reply plutus-frontend/src/recommendations.js]: junk, i removed this and redid the pr

**Comment by bestnewkevin** [comment]
Screen Shot 2019-12-14 at 7 56 23 PM

File: plutus-frontend/src/recommendations.js (hunk 1 above)
Comment (dnntrn) [reply plutus-frontend/src/recommendations.js]: lol coolio, but you can also just make changes, git add, git commit, and git push without closing the PR. it automatically updates the PR. That's one of the reasons to open a PR first in development. Makes things faster too!

File: plutus-frontend/src/components/FullChart.js (hunk 2 above)
Comment (bestnewkevin) [reply plutus-frontend/src/components/FullChart.js]: im passing in the names of the company as well as the levels through props, gauresh and i will clean this up when we figure out putting jsons in there

File: plutus-frontend/src/recommendations.js (hunk 1 above)
Comment (bestnewkevin) [reply plutus-frontend/src/recommendations.js]: github is godhub

**Comment by dnntrn** [comment]
awesome! 🎉

**Pull request merged**